data/ecosystem/adopters.yaml

```diff
@@ -127,6 +129,7 @@
   url: https://www.vtex.com/
   components: [Collector, Go, JavaScript, .NET]
   reference: 'https://colocatedeventseu2023.sched.com/event/1Jo8E/ingesting-65-tb-of-telemetry-data-daily-through-open-telemetry-protocol-and-collectors-gustavo-pantuza-vtex'
+  referenceTitle: blog post
```

Comment:
that's not a blog post, I would say "presentation" or "talk" or similar

Reply:
should i refer it as "Event post"?

Reply:
I think "presentation" is the best title, since we have this for another example already

Reply:
ok i'll correct that as presentation